<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Database</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Table" name="tableView">
      <property name="minimumSize">
       <size>
        <width>1100</width>
        <height>560</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionNewWindow"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
    <addaction name="actionDeutsch"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionSelection"/>
    <addaction name="actionSearch"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionDiagram"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFilter"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMenu_bar_on_off"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSearch"/>
   <addaction name="separator"/>
   <addaction name="actionSelection"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionDiagram"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Open.png</normaloff>:/Icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Add.png</normaloff>:/Icons/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="statusTip">
    <string>Add a new note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Edit.png</normaloff>:/Icons/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="statusTip">
    <string>Edit current note</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Delete.png</normaloff>:/Icons/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete current note</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Save2.png</normaloff>:/Icons/Save2.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Quick save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>NewWindow</string>
   </property>
  </action>
  <action name="actionMenu_bar_on_off">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Hamburger_icon.svg.png</normaloff>:/Icons/Hamburger_icon.svg.png</iconset>
   </property>
   <property name="text">
    <string>Menu bar on/off</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Refresh.png</normaloff>:/Icons/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string> Reset filters</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Selection.png</normaloff>:/Icons/Selection.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="statusTip">
    <string>Filter on parameteres</string>
   </property>
  </action>
  <action name="actionDiagram">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Diagram.png</normaloff>:/Icons/Diagram.png</iconset>
   </property>
   <property name="text">
    <string>Diagram</string>
   </property>
   <property name="statusTip">
    <string>Open Diagram window</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/Search.png</normaloff>:/Icons/Search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="statusTip">
    <string>Search by string</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/Icons/SaveAs.png</normaloff>:/Icons/SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="statusTip">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Table</class>
   <extends>QTableView</extends>
   <header>table.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
